packages/babel-plugin-transform-async-to-generator/test/fixtures/assumption-ignoreFunctionLength-true/basic/input.js

```diff
@@ -0,0 +1,7 @@
+async (x = 1) => 1;
+(async function (x = 2) {
+  return 2;
+})();
+
+foo(async (...x) => {});
+foo(async (...[...y]) => {});
```

Comment:
All these functions would have `length: 0` anyway: to test the behavior of `ignoreFunctionLength`, we shuold use functions with `.length >= 1`.

Reply:
Thanks.
I was writing on the first case and accidentally mixed it up.
I will update this test case.